eo-phi-normalizer/test/eo/phi/rules/yegor.yaml

```diff
@@ -6,9 +6,9 @@ rules:
       global_object: '!Phi'
       current_object: '⟦ !a ↦ ⟦ !B ⟧, !B_rest ⟧'
     pattern: |
-      ⟦ !B ⟧
+      ⟦ !a ↦ ⟦ !B ⟧, !B_rest ⟧
```

Comment:
But then it becomes the same as `current_object` (and also what it was before adding `context`). I don't remember the exact example, but I believe there were test cases that this didn't match

Reply:
Why can't you? That's how it works. The `current_object` itself contains meta variables, so even after replacing the pattern will still have metavariables that can be matched with `obj`

Reply:
I remembered the test case that this breaks: `⟦ φ ↦ ⟦ φ ↦ ⟦ ⟧ ⟧.σ ⟧`
The issue is that Rule 8 applies because it think that `⟦ φ ↦ ⟦ ⟧ ⟧.σ` is in normal form because rule 1 does not apply to just an empty formation (the inner `⟦ ⟧` in this case), only to nested ones, but `⟦ !a ↦ ⟦ !B ⟧, !B_rest ⟧` doesn't match because we have object dispatch outside rather than just a formation